File: tests/fixtures/javascript/data_flow.js
function transform(input) {
    let x = input;
    let y = x;
    return y;
}

function multiAssign(a, b) {
    let result = a;
    result = b;
    return result;
}

const value = transform("hello");
const output = multiAssign(value, "world");

File: tests/fixtures/go/data_flow.go
package main

func transform(input string) string {
	x := input
	y := x
	return y
}

func multiAssign(a, b string) string {
	result := a
	result = b
	return result
}

func multiReturn() (string, int) {
	return "hello", 42
}

func useMultiReturn() {
	s, n := multiReturn()
	_ = s
	_ = n
}
